src/main/java/se/sundsvall/installation/api/model/InstallationsResponse.java

```diff
@@ -0,0 +1,31 @@
+package se.sundsvall.installation.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+import java.util.List;
+
+import generated.se.sundsvall.datawarehousereader.PagingAndSortingMetaData;
```

Comment:
DWR-class that should be replaced to decouple the facade class from the backend service. It's better to use the PagingMetaData class in Dept-44. See how it's done in api-service-document for example.
```
import se.sundsvall.dept44.models.api.paging.PagingMetaData;

@Schema(description = "Paged document response model", accessMode = READ_ONLY)
public class PagedDocumentResponse {

	@ArraySchema(schema = @Schema(implementation = Document.class))
	private List<Document> documents;

	@JsonProperty("_meta")
	@Schema(implementation = PagingMetaData.class)
	private PagingMetaData metadata;

(getters and setters omitted for brevity)
```

Reply:
Used the wrong import..